src/sentry/static/sentry/app/views/settings/account/apiTokens.jsx

```diff
@@ -76,6 +79,14 @@ class ApiTokens extends AsyncView {
     return (
       <div>
         <SettingsPageHeader title="Auth Tokens" action={action} />
+        <AlertLink
+          to={`/settings/${organization.slug}/developer-settings/`}
+          icon="icon-circle-info"
+        >
+          {t(
+            'Auth Tokens are tied to the logged in user. If the user leaves the organization, the token will no longer work! We suggest using internal integrations to manage tokens which are not tied to a user.'
```

Comment:
nit: i would probably say 'which are tied to the organization' instead of 'which are not tied to a user'

Reply:
actually that sounds lil weird maybe like 'to manage tokens that are tied to the organization instead'. or something. basically what I'm trying to get at is explaining the organization relationship. anyway not a blocker so

Reply:
Reworded it @MeredithAnya, let me know what you think 👍